db/migrate/20230628092235_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :description
+      t.string :comments_counter
+      t.integer :likes_counter
+      t.timestamps
```

Comment:
- [ ] As the previous reviewer said `comment_counter` and `like_counter` should be integers but I can see that you have applied it only on `likes_counter`. Please apply it also on `comments_counter`.